services/crates/crates-user-downloads.tester.js

```diff
@@ -0,0 +1,13 @@
+import { isMetric } from '../test-validators.js'
+import { createServiceTester } from '../tester.js'
+export const t = await createServiceTester()
+
+t.create('total user downloads')
+  .get('/udt/3027.json')
+  .expectBadge({ label: 'downloads', message: isMetric })
+
+// we can not test for id not linked to user as it returns 0 downloads
```

Comment:
I reckon we should put in a test case for a non-existant user id anyway, `.expectBadge({ label: 'downloads', message: '0' })` in the assertion and explain in a comment that this API returns `{"total_downloads": 0}` with a `200 OK` instead of a 404 response for a not found user id.

Reply:
Added the test with 230c79aa9d24d903a879586c6b43afed216af0a8.
I realized they use int 32 for userid so i picked 2^32 as its the last id which has the least chance of getting taken.